src/Microsoft.AspNet.Razor/TagHelpers/TagHelperDescriptor.cs

```diff
@@ -136,6 +141,11 @@ public class TagHelperDescriptor
         /// <remarks>
         /// <c>*</c> at the end of an attribute name acts as a prefix match.
         /// </remarks>
-        public IReadOnlyList<string> RequiredAttributes { get; private set; }
+        public IReadOnlyList<string> RequiredAttributes { get; }
+
+        /// <summary>
+        /// The <see cref="TagHelperUsageDescriptor"/> that indicates how to use the tag helper at design time.
```

Comment:
Stick with the "contains information about" wording for all `TagHelperUsageDescriptor` properties and parameters.